src/netbase.h

```diff
@@ -25,6 +26,22 @@ static const int DEFAULT_CONNECT_TIMEOUT = 5000;
 //! -dns default
 static const int DEFAULT_NAME_LOOKUP = true;
 
+enum class ConnectionDirection {
+        None = 0,
+        In = (1U << 0),
+        Out = (1U << 1),
+        Both = (In | Out),
```

Comment:
Any reason the new code is not clang formatted?

Reply:
I don't see any documented way to do that (`contrib/devtools/clang-format-diff.py` wants to make a bunch of unrelated changes).

Fixed indentation. Anything else?